easypost/carrier_account.py

```diff
@@ -16,6 +18,14 @@
 )
 
 
+def _select_carrier_account_creation_endpoint(carrier_account_type: Optional[Any]) -> str:
+    """Determines which API endpoint to use for the creation call."""
+    if carrier_account_type in _CARRIER_ACCOUNT_TYPES_WITH_CUSTOM_WORKFLOWS:
```

Comment:
Done to suggest it's internal (like a private method)

Reply:
Right, I understand why but meh for constants.

They no longer match. My suggestion didn't have the underscore in it so you'll need to adjust one or the other to match since this var is now undefined.